src/assimp/assimp.cpp

```diff
@@ -726,6 +786,8 @@ assimp::Implementation::BindState assimp::Implementation::processMaterials(const
             // Phong shading
             vsg::PhongMaterial mat;
 
+            if (alphaBlend)
+                mat.alphaMask = 0.0f;
```

Comment:
What is the motivation for setting the mask to 0.0f?  You may still want to use the mask when alpha blending,

Reply:
Usually you either want mask or blending. At least in most of game engines that I have worked with those were separate shaders/materials. The reason is that discard is bad for early-z and very bad for tile based hardware.